pkgs/tools/misc/calamares/default.nix

```diff
@@ -37,10 +37,14 @@ mkDerivation rec {
         -i com.github.calamares.calamares.policy
 
     sed -e 's,/usr/share/zoneinfo,${tzdata}/share/zoneinfo,' \
-        -i src/modules/locale/SetTimezoneJob.cpp
+        -i src/modules/locale/SetTimezoneJob.cpp \
+        -i src/modules/locale/Tests.cpp \
+        -i src/libcalamares/locale/TimeZone.cpp \
+        -i src/libcalamares/locale/zone-extractor.py
 
     sed -e 's,/usr/share/X11/xkb/rules/base.lst,${xkeyboard_config}/share/X11/xkb/rules/base.lst,' \
-        -i src/modules/keyboard/keyboardwidget/keyboardglobal.h
+        -i src/modules/keyboard/keyboardwidget/keyboardglobal.cpp \
+        -i src/modules/keyboard/layout-extractor.py
```

Comment:
Not familiar with the codebase, but I think this file is only used to generate [`src/modules/keyboard/KeyboardData_p.cxxtr`](https://github.com/calamares/calamares/blob/calamares/src/modules/keyboard/KeyboardData_p.cxxtr) and maybe we don't need to modify it.